src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,3 +1,32 @@
+import React from 'react';
+import { UserInfo } from '../UserInfo';
+
+interface Props {
+  preparedTodo: Todo,
+}
+
+export const TodoInfo: React.FC<Props> = ({ preparedTodo }) => {
+  const {
+    title,
+    completed,
+    user,
+  } = preparedTodo;
+
+  return (
```

Comment:
i might be missing something, but you should be able to de-structure right inside params and use an implicit return
```suggestion
export const TodoInfo: React.FC<Props> = ({
    title,
    completed,
    user,
  }) => (
```